crates/routes/src/utils/scheduled_tasks.rs

```diff
@@ -363,6 +370,59 @@ async fn active_counts(pool: &mut DbPool<'_>) -> LemmyResult<()> {
   Ok(())
 }
 
+async fn local_site_stats(pool: &mut DbPool<'_>) -> LemmyResult<()> {
+  info!("Updating local site stats ...");
+
+  let mut conn = get_conn(pool).await?;
+
+  let user_count: i64 = local_user::table
```

Comment:
All this below should be moved to an impl function in `crates/db_schema/src/impls/local_site.rs` . 

Then you could probably even keep the tests above by calling it.